integration/tests/grid-tooltip.test.js

```diff
@@ -209,6 +211,39 @@ describe('tooltip', () => {
     expect(tooltip.opened).to.be.false;
   });
 
+  it('should hide the tooltip when starting scrolling', async () => {
```

Comment:
This also solves an issue in Safari which does not trigger `mouseenter` while scrolling (with a trackpad at least):

https://user-images.githubusercontent.com/357820/197718406-f8b66f5c-9ad3-4569-9782-4b7c40a81815.mov